lua/downloader/resource_scanner.lua

```diff
@@ -31,7 +31,7 @@ end
 function MODULE:Run(context)
     for _, addon in ipairs(context.addons) do
         if not table.HasValue(context.ignoreResources, addon.wsid) then
-            local hasResources = addon.models > 0 or HasUsingResource("", addon.title)
+            local hasResources = context.cache[tonumber(addon.wsid)] or addon.models > 0 or HasUsingResource("", addon.title)
```

Comment:
Use context.ignoreResources instead, it exists exactly for that

Reply:
I don't want to ignore the addon, I want to avoid a an unnecessary scan. This way we just add a entry into context.usingAddons and load it later

Reply:
This is exactly what context.ignoreResources does

Reply:
You have to add the addon to usingAddons and ignoreResources